v1/topdown/save.go

```diff
@@ -470,3 +512,18 @@ func (i *inliningControl) Disabled(ref ast.Ref, ignoreInternal bool) bool {
 	}
 	return false
 }
+
+func (i *inliningControl) DisabledVar(v ast.Var, ignoreInternal bool) bool {
+	if i == nil {
+		return false
+	}
+
+	for _, frame := range i.disable {
+		if !frame.internal || !ignoreInternal {
+			if frame.v.Equal(v) {
+				return true
+			}
+		}
```

Comment:
🤔 
```suggestion
		if !frame.internal || !ignoreInternal && frame.v.Equal(v) {
			return true
		}
```
...or is it not the same?